spec/uo_cli_spec.rb

```diff
@@ -346,6 +346,13 @@ def delete_directory_or_file(dir_or_file)
       expect(File.exist?(File.join(test_directory, 'run', 'reopt_scenario', 'process_status.json'))).to be true
     end
 
+    it 'reopt post-processes a scenario with resilience reporting' do
+      system("#{call_cli} process --reopt-scenario --reopt-resilience --scenario #{test_reopt_scenario} --feature #{test_feature}")
+      expect(File.exist?(File.join(test_directory, 'run', 'reopt_scenario', 'scenario_optimization.json'))).to be true
+      expect(File.exist?(File.join(test_directory, 'run', 'reopt_scenario', 'process_status.json'))).to be true
+      expect(File.exist?(File.join(test_directory, 'run', 'reopt_scenario', 'reopt', 'scenario_report_reopt_scenario_reopt_run_resilience.json'))).to be true
+    end
+
```

Comment:
if it's easy, you could also check for one of these keys in the generated report: 
https://github.com/urbanopt/urbanopt-reopt-gem/blob/49993547fabc5431fffa7e534908a8a7b01182e2/lib/urbanopt/reopt/scenario_report_adapter.rb#L242-L249